bin/coveralls

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env php
 <?php
 
-if (file_exists(__DIR__ . '/../../vendor/autoload.php')) {
-    require __DIR__ . '/../../vendor/autoload.php';
+if (file_exists(__DIR__ . '/../vendor/autoload.php')) {
```

Comment:
should this be changed?
https://github.com/satooshi/php-coveralls/pull/135#issuecomment-162943298